src/main/resources/locales/en/translation/claim-saved.json

```diff
@@ -0,0 +1,3 @@
+{
+    "continueWithClaim": "Continue with your claim"
```

Comment:
The 'Continue with your claim' could be moved into the json translation file also.